desktop/README.md

```diff
@@ -27,7 +27,9 @@ poetry install
 
 ### Get Tor
 
-**Linux users:** You may need the `libxcb-xinerama0` and `libxcb-cursor0` packages installed.
+**Linux users:** 
+- On Debian/Debian-based distributions you may need the `libxcb-xinerama0` and `libxcb-cursor0` packages installed.
+- On Fedora/CentOS you may need the `libxcb-1.13.1-11.fc38.x86_64` and `xcb-util-0.4.1-2.fc38.x86_64` packages installed.
```

Comment:
Just to 'future-proof' this, I'd recommend we remove the version-specific numbers, and just put `libxcb` and `xcb-util` here, but otherwise, great!